src/MPI.jl

```diff
@@ -53,29 +55,32 @@ include("collective.jl")
 include("topology.jl")
 include("onesided.jl")
 include("io.jl")
-include("errorhandler.jl")
+include("errhandler.jl")
 include("mpiexec_wrapper.jl")
 
 include("deprecated.jl")
 
 function __init__()
 
     @static if Sys.isunix()
-        # need to open libmpi with RTLD_GLOBAL flag for Linux, before
-        # any ccall cannot use RTLD_DEEPBIND; this leads to segfaults
-        # at least on Ubuntu 15.10
+        # - need to open libmpi with RTLD_GLOBAL flag for Linux,
+        #   before any ccall
+        # - cannot use RTLD_DEEPBIND; this leads to segfaults at least
+        #   on Ubuntu 15.10
         Libdl.dlopen(libmpi, Libdl.RTLD_LAZY | Libdl.RTLD_GLOBAL)
     end
 
     __init__deps()
 
+    include(joinpath(dirname(@__DIR__), "src", "read_load_time_mpi_constants.jl"))
```

Comment:
```suggestion
    include("read_load_time_mpi_constants.jl")
```

Reply:
This leads to
```
ERROR: LoadError: InitError: SystemError: opening file "/Users/eschnett/.julia/dev/MPI/test/read_load_time_mpi_constants.jl": No such file or directory
```
during `Pkg.test()`.

Reply:
Oh, this is being included at runtime. Won't that then break if you run it from a different directory? At the very least it will make precompilation difficult.

Why not put the contents of `read_load_time_mpi_constants.jl` in a function, and call that?

Reply:
This is now a function.